errors/errors.rb

```diff
@@ -1,20 +1,21 @@
-# frozen_string_literal: true
 
-module Errors # :nodoc:
-  class WrongClassError < StandardError # :nodoc:
+
+module Errors
+  class WrongClassError < StandardError
     def initialize
       super('Wrong class!')
     end
   end
 
-  class EmptyStringError < StandardError # :nodoc:
+  class EmptyStringError < StandardError
     def initialize
       super('You try to send empty string!')
     end
   end
 
-  class UndefinedLibraryEntity < StandardError # :nodoc:
+  class UndefinedLibraryEntity < StandardError
     def initialize
+      'qweqwe'.gsub('q','s')
```

Comment:
Using tr is faster than gsub when replacing a single character in a string with another single character